### ISSUE TITLE
Log llama.cpp commit hash when installing using --verbose

### ISSUE THREAD

**Issue by gjmulder** [abetlen/llama-cpp-python#256]
See https://github.com/abetlen/llama-cpp-python/issues/223#issuecomment-1556207716

The bug looks to be with `libllama`, but in order to log the bug with [llama.cpp](https://github.com/ggerganov/llama.cpp) we ideally need to specify which commit hash of `libllama` the bug occurs in.

**Comment by abetlen** [comment]
Makes sense, there should be a `build-info.h` file generated inside the `vendor/llama.cpp` folder so we should be able to use that, I'll write a `get_version` function in the `llama_cpp.py` file to pull that in.
